doc/src/devdocs/init.md

```diff
@@ -156,21 +156,18 @@ executes it.
 
 ## `Base._start`
 
-[`Base._start`](https://github.com/JuliaLang/julia/blob/master/base/client.jl) calls [`Base.process_options`](https://github.com/JuliaLang/julia/blob/master/base/client.jl)
+[`Base._start`](https://github.com/JuliaLang/julia/blob/master/base/client.jl) calls [`Base.exec_options`](https://github.com/JuliaLang/julia/blob/master/base/client.jl)
 which calls [`jl_parse_input_line("println("Hello World!")")`](https://github.com/JuliaLang/julia/blob/master/src/ast.c)
-to create an expression object and [`Base.eval()`](@ref eval) to execute it.
+to create an expression object and [`Core.eval(Main, ex)`](@ref Core.eval) to execute the parsed expression `ex` in the module context of `Main`.
 
-## `Base.eval`
+## `Core.eval`
```

Comment:
FWIW, the previous `Base.eval` section creates a false documentation binding `@ref Base.eval`, which are being used in several other places, which broke the documentation system on this PR.

They should point to `Base.MainInclude.eval` instead, so I fixed all the references.

Once the CI passes, this should be good to go.

Reply:
Thank you. It is interesting that CI checks documents also.